src/zajecia10/zadanie3/NamesHolder.java

```diff
@@ -0,0 +1,51 @@
+package zajecia10.zadanie3;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class NamesHolder {
+    List<String> list = new ArrayList<>();
```

Comment:
to bym zrobil prywatne. Wtym xzadaniu specjalnie chowamy liste (ablo tablice) a wystawiamy tylko nasze publiczne metody

Reply:
zmienione